app/src/main/java/live/hms/app2/ui/meeting/MeetingFragment.kt

```diff
@@ -411,25 +411,20 @@ class MeetingFragment : Fragment() {
       }
     }
 
-    viewLifecycleOwner.lifecycleScope.launch {
-      meetingViewModel.rtmpErrors.collect { rtmpException ->
-        if (rtmpException != null) {
-          withContext(Dispatchers.Main) {
-            Toast.makeText(context, "RTMP error $rtmpException", Toast.LENGTH_LONG).show()
+    lifecycleScope.launch {
```

Comment:
The entire lifecycle of the fragment extends far beyond the view and I'd want this to be stopped as soon as the view is destroyed so it's important that the operations be carried out on the viewLifeCycleOwner.

```suggestion
    viewLifecycleOwner.lifecycleScope.launch {
```